CHANGELOG.md

```diff
@@ -8,6 +8,16 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 _No documentation available about unreleased changes as of yet._
 
+## [2.0.7] - 2019-11-28
```

Comment:
you are missing a link at a bottom of this page

Reply:
Huh, didn't know you could a) define links and then reference them by anchor and b) that we have a handy link to the diff. 👌